### PR TITLE
21w06a

### PR DESCRIPTION
**Pull request by ghost**
**TODO:**

- [x] Add back holiday hat
- [x] Figure out how the StackToolTipHandler can spawn a BlockEntity without pos and state
- [x] Somehow make UnloadHandler functional again

### CODE REVIEW

File: src/main/java/reborncore/mixin/common/MixinWorld.java [hunk 1]
```diff
@@ -39,7 +39,7 @@
 @Mixin(World.class)
 public class MixinWorld {
 
-	@Shadow
+	/* FIXME @Shadow
```
Comment (modmuss50) [inline src/main/java/reborncore/mixin/common/MixinWorld.java]: There is a fabric event for this now. Should prob move over to it in the 1.16 branch and then pull that change in here.

File: src/main/java/reborncore/mixin/client/MixinPlayerEntityRenderer.java [hunk 2]
```diff
@@ -26,29 +26,31 @@
...
 public abstract class MixinPlayerEntityRenderer extends LivingEntityRenderer<AbstractClientPlayerEntity, PlayerEntityModel<AbstractClientPlayerEntity>> {
 
-	public MixinPlayerEntityRenderer(EntityRenderDispatcher entityRenderDispatcher_1, PlayerEntityModel<AbstractClientPlayerEntity> entityModel_1, float float_1) {
-		super(entityRenderDispatcher_1, entityModel_1, float_1);
+	public MixinPlayerEntityRenderer(EntityRendererFactory.Context ctx, PlayerEntityModel model, float shadowRadius) {
+		super(ctx, model, shadowRadius);
 	}
 
-	@Inject(method = "<init>(Lnet/minecraft/client/render/entity/EntityRenderDispatcher;Z)V", at = @At("RETURN"))
+	/*@Inject(method = "<init>(Lnet/minecraft/client/render/entity/EntityRenderDispatcher;Z)V", at = @At("RETURN"))
```
Comment (modmuss50) [inline src/main/java/reborncore/mixin/client/MixinPlayerEntityRenderer.java]: Again here, fabric now has an event for this. Im also not against removing this easter egg as its getting kinda stale :D

**Review by modmuss50 — COMMENTED** [review]
Looks like a good start. 👍

File: src/main/java/reborncore/mixin/common/MixinWorld.java (hunk 1 above)
Comment (ghost) [reply src/main/java/reborncore/mixin/common/MixinWorld.java]: Rebased on 1.16

File: src/main/java/reborncore/mixin/client/MixinPlayerEntityRenderer.java (hunk 2 above)
Comment (ghost) [reply src/main/java/reborncore/mixin/client/MixinPlayerEntityRenderer.java]: Rebased on 1.16 and re-added holiday hat

File: build.gradle [hunk 3]
```diff
@@ -109,7 +109,7 @@ curseforge {
 		id = "237903"
 		changelog = "A changelog can be found at https://github.com/TechReborn/RebornCore"
 		releaseType = ENV.RELEASE_CHANNEL ?: "release"
-		addGameVersion "1.16.5"
+		addGameVersion "21w06a"
```
Comment (modmuss50) [inline build.gradle:112]: Should be '1.17' not a deal breaker for now.

**Review by modmuss50 — COMMENTED** [review]
Looks like a great start, ill pull this in, will be easier to keep track of further changes that way.

**Pull request merged**